added missing release notes entries (#116)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/applications-10-16-0-bundle/10-16-0-0.md

<tr>
<td>Cockpit</td>
<td>In the data explorer, the <b>Create smart rule</b> option has been added back to the context menu for datapoint selection list items.</td>
<td>MTM-49612</td>
</tr>

<tr>
<td>Cockpit</td>

> <td>In HTML and SVG widget configurations, the sanitization options "None", "Lax" and "Strict" are now selectable.</td>

Author: ```suggestion
<td>In HTML and SVG widget configurations, you can now select the code sanitization options "None", "Lax" and "Strict".</td>

```

<tr>
<td>Cockpit</td>
<td>In the data explorer, the <b>Create smart rule</b> option has been added back to the context menu for datapoint selection list items.</td>
<td>MTM-49612</td>
</tr>

<tr>
<td>Cockpit</td>
<td>In HTML and SVG widget configurations, you can now select the code sanitization options "None", "Lax" and "Strict".</td>

> 

Reviewer: ```suggestion

```
